app/components/common/Navbar.js

```diff
@@ -1,23 +1,20 @@
-import React from 'react'
+import React, {Component} from 'react'
+import ConnectionForm from '../connectionForm';
+import Indicator from '../indicator';
 
-export default class Navbar extends React.Component {
+export default class Navbar extends Component {
   render() {
     return(
       <nav className="navbar navbar-default navbar-fixed-top">
         <div className="container">
           <div className="navbar-header">
-            <a className="navbar-brand" href="#">Project name</a>
+            <a className="navbar-brand" href="#">Rbkit Client</a>
```

Comment:
This should be just "Rbkit" . "Rbkit client" and "Rbkit Server" are just internal nomenclature